kuma/wiki/models.py

```diff
@@ -1599,9 +1599,8 @@ class DocumentZone(models.Model):
     """
     document = models.OneToOneField(Document, related_name='zone')
     css_filename = models.CharField(
-        max_length=100, null=True, blank=True,
-        help_text="name of an alternative CSS file for documents under this "
-                  "zone")
+        max_length=100, blank=True, help_text="name of an alternative CSS file "
+                                              "for documents under this zone")
     url_root = models.CharField(
         max_length=255, null=True, blank=True, db_index=True,
```

Comment:
Yeah, they did this one wrong.  Don't trust existing code!

_checks blame to make sure he didn't do it_

Don't trust it!